docs/user/resources/40-50-20-gcp-redis-cluster.md

```diff
@@ -0,0 +1,78 @@
+# GcpRedisCluster Custom Resource
+
+The `gcprediscluster.cloud-resources.kyma-project.io` is a namespace-scoped custom resource (CR).
+
+It describes the [Google Memorystore Redis Cluster](https://cloud.google.com/memorystore/docs/cluster/memorystore-for-redis-cluster-overview) instance.
+
+Once the instance is provisioned, a Kubernetes Secret with endpoint and credential details is provided in the same namespace.
+By default, the created auth Secret has the same name as the GcpRedisCluster, unless specified otherwise.
+
+Redis Cluster requires a range of IP Addresses. They can be allocated using the GcpSubnet CR.
+If a GcpSubnet CR is not specified in the GcpRedisCluster, then the default GcpSubnet is used.
+If the default GcpSubnet does not exist, it is automatically created.
+Manually create a non-default GcpSubnet with specified Classless Inter-Domain Routing (CIDR) and use it only in advanced cases of network topology when you want to control the network segments to avoid range conflicts with other networks.
```

Comment:
```suggestion
You can manually create a non-default GcpSubnet with specified Classless Inter-Domain Routing (CIDR) and use it only in advanced cases of network topology when you want to control the network segments to avoid range conflicts with other networks.
```